### PR TITLE
Safe parts of the refactor branch

### PR DESCRIPTION
**Pull request by krlmlr**
No own commits, includes refactor branch minus:

1. Removal of `create_bindings()` and `delete_bindings()`,
2. Removal of `codetools` (which currently is incompatible with active bindings).

I can submit a PR for 2. once this is merged. Will also resubmit #102 then.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: remake
 Title: Make-like build management
-Version: 0.2.0
+Version: 0.3.0
```
Comment (krlmlr) [inline DESCRIPTION]: Which versioning scheme are you using?

File: R/utils.R [hunk 2]
```diff
@@ -188,7 +188,7 @@ path_split <- function(x) {
 
 file_copy <- function(from, to, ..., warn=TRUE) {
   assert_scalar_character(from)
-  ok <- file.exists(from) && file.copy(from, to)
+  ok <- file.exists(from) && file.copy(from, to, TRUE)
```
Comment (krlmlr) [inline R/utils.R]: `overwrite = TRUE`

**Comment by krlmlr** [comment]
Tests now passing on my system.

**Comment by krlmlr** [comment]
...and on Travis (but not yet on Windows, I can take a look at them soon).

**Comment by richfitz** [comment]
This is great, thanks!

**Comment by krlmlr** [comment]
Would you mind merging this? New PRs will depend on these changes.

**Pull request merged**